maya/mel/assign-pbr-textures.mel

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Might be a good idea to place this code in a MEL procedure? E.g. `assign_glTF_PBR_material_and_textures`? Passing the node id as a parameter, using the selection if it is the empty string?